lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -92,6 +99,18 @@
     timezone: America/Chicago
     tags: production, chicago
   delegate_to: localhost
+- name: Create combined network named MyNet in the YourOrg organization
+  meraki_network:
+    auth_key: abc12345
+    state: present
+    org_name: YourOrg
+    net_name: MyNet
+    types:
+      - switch
+      - appliance
+    timezone: America/Chicago
+    tags: production, chicago
```

Comment:
This really should be defined as a list.
And that's good news, because it supports comma-separated string values automatically !

Reply:
So are you good with how it is?